### PR TITLE
feat(opensea): compatible schema

### PR DESCRIPTION
**Pull request by gnarco**

### CODE REVIEW

**Review by mouadhbb — APPROVED** [review]

File: public/version5/ArianeeProductCertificate-i18n.json [hunk 1]
```diff
@@ -0,0 +1,1546 @@
+{
+  "$id": "https://cert.arianee.org/version4/ArianeeProductCertificate-i18n.json",
```
Comment (stefdelec) [inline public/version5/ArianeeProductCertificate-i18n.json:2]: version5!!!

File: public/version5/ArianeeProductCertificate-i18n.json [hunk 2]
```diff
@@ -0,0 +1,1546 @@
...
+  "$schema": "https://cert.arianee.org/version4/ArianeeProductCertificate-i18n.json",
```
Comment (stefdelec) [inline public/version5/ArianeeProductCertificate-i18n.json:3]: version5!!! (2fois)

File: public/version5/ArianeeProductCertificate-i18n.json [hunk 3]
```diff
@@ -0,0 +1,1546 @@
...
+  "title": "Arianee Certificate i18n (transparency and composed)",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "title": "$schema",
+      "type": "string",
+      "default": "https://cert.arianee.org/version4/ArianeeProductCertificate-i18n.json",
```
Comment (stefdelec) [inline public/version5/ArianeeProductCertificate-i18n.json:10]: v5

**Review by stefdelec — DISMISSED** [review]

File: public/version5/ArianeeProductCertificate-i18n.json (hunk 1 above)
Comment (gnarco) [reply public/version5/ArianeeProductCertificate-i18n.json]: done

**Pull request merged**